server/views/yes-no-idk.html

```diff
@@ -22,9 +22,6 @@
 
     <div class="govuk-grid-column-two-thirds">
       <form method="post">
-
-        <textarea name="pageName" hidden>{{ pageName }}</textarea>
-
         {{ govukRadios({
           idPrefix: "ivoryAdded",
           name: "ivoryAdded",
```

Comment:
Maybe if we are going to use this as a generic yes/no/idk page we should update lines 17, 26 & 27 to something more generic like yesNoIdk?